fw/http.c

```diff
@@ -1093,6 +1093,7 @@ tfw_h2_resp_fwd(TfwHttpResp *resp)
 	}
 	else {
 		TFW_INC_STAT_BH(serv.msgs_forwarded);
+		tfw_inc_global_hm_stats(resp->status);
```

Comment:
Why we call this function here and don't call it in other places where `TFW_INC_STAT_BH(serv.msgs_forwarded);` is called? For example `__tfw_http_resp_fwd`.